README.md

```diff
@@ -128,18 +127,17 @@ fn run() -> Result<(), Error> {
  passing it to `Elasticsearch::new()` create a new instance of `Elasticsearch`
 
 ```rust,no_run
+use url::Url;
 use elasticsearch::{
-    auth::Credentials,
-    Error, Elasticsearch, 
+    Error, Elasticsearch,
     http::transport::{TransportBuilder,SingleNodeConnectionPool},
 };
-use url::Url;
 
-fn run() -> Result<(), Error> {
+fn main() -> Result<(), Error> {
     let url = Url::parse("https://example.com")?;
     let conn_pool = SingleNodeConnectionPool::new(url);
     let transport = TransportBuilder::new(conn_pool).disable_proxy().build()?;
-    let client = Elasticsearch::new(transport);
+    let _client = Elasticsearch::new(transport);
```

Comment:
I realize client isn’t used, but would prefer to keep this as `client`, in line with all examples.